### ISSUE TITLE
How to search for stations outside of dresden

### ISSUE THREAD

**Issue by lefty78** [kiliankoe/dvbjs#35]
Hello,

I wnat to search for locations outside of Dresden, for example Zinzendorfstrasse (Radebeul). But i dont get any info back. 

What im doing wrong?

**Comment by kiliankoe** [comment]
Seems to be due to https://github.com/kiliankoe/dvbjs/blob/master/lib/find.js#L34, which limits the results to having 'Dresden' as a region. I forget why that's in there, @ronhippler @jens-maus probably know why^^

**Comment by jens-maus** [comment]
Well, this isn't actually a limitation of the new webapi based find() version but was also find() uses with the old version were limited to results of Dresden only. However, the request seems fine but this would require a slightly changed number of paramters for the find() function where one can specify the city of the find request as well. Or find() should be changed to deal with more than one result and thus supply the city information as well.

**Comment by kiliankoe** [comment]
The following should actually work fine with the current implementation, I tried it earlier. And it does if I comment out the filtering of results that don't have Dresden as a region.

```js
dvb.find('zinzendorfstr radebeul', function(err, data) {
    if (err) { console.log(err) }
    console.log(JSON.stringify(data, null, 4));
})
```

I don't believe an extra parameter for the city is necessary, the pointfinder endpoint handles that quite well. I'm just not sure why the filtering is in there. If that doesn't have any valid reason it might as well be removed.

**Comment by jens-maus** [comment]
@kiliankoe Just try to do a search for "Körnerplatz" without the fitering for Dresden and your will see that point finder will return more than one result which IMHO should then also be properly dealt with but find() doesn't support more than one result yet.

**Comment by kiliankoe** [comment]
Don't quite understand what you mean. 

I get this repsonse:

```js
{
  "PointStatus": "List",
  "Status": {
    "Code": "Ok"
  },
  "Points": [
    "33000832|||Körnerplatz|5659233|4627336|0||",
    "36080103||Döbeln|Körnerplatz|5665477|4578394|0||",
    "36080155||Döbeln|Körnerplatz-Schule|5665360|4578413|0||"
  ],
  "ExpirationTime": "\/Date(1488157247000+0100)\/"
}
```

And this result via dvbjs with the filtering removed:

```js
[
    {
        "stop": "Körnerplatz",
        "id": "33000832",
        "coords": [
            51.05411984235784,
            13.814664768428814
        ]
    },
    {
        "stop": "Körnerplatz",
        "id": "36080103",
        "coords": [
            51.1189877898748,
            13.118164070373632
        ]
    },
    {
        "stop": "Körnerplatz-Schule",
        "id": "36080155",
        "coords": [
            51.117933714936974,
            13.118409987591614
        ]
    }
]
```

The region should probably be thrown in there as well with an empty string probably being Dresden.

**Comment by jens-maus** [comment]
@kiliankoe Sorry, but that's exactly what I meant. Currently there is no possibility to distinguish the city in the results, thus the result json should also contain the city/region. And yes, if it is empty it seems to be Dresden, which IMHO should the be manually added to the result json of the find() function.

**Comment by kiliankoe** [comment]
Ah ok, so we're thinking in the same direction :)

Looks like the filter was introduced in 0c64592cdd17fed3c44a42e6a051503148ebeae7, do you happen to know why? If it's not necessary for some reason, why don't we just remove it altogether and throw the region into the response objects as well.

**Comment by jens-maus** [comment]
@kiliankoe I introduced it because of this missing possibility to distinguish between city and because I saw `place_sf: Dresden` in https://github.com/kiliankoe/dvbjs/commit/0c64592cdd17fed3c44a42e6a051503148ebeae7#diff-482f58edb0405242efa18dadc5e25da2L20 and thought that this also filtered by Dresden results only. However, thinking about it once more we should then indeed revise the filtering and return all results but with `"city": "Dresden"` like results being added to the output JSON data.

**Comment by kiliankoe** [comment]
Just released v4.3.3 that should fix this. Thanks for reporting it, @lefty78!
